graph/encoding/dot/decode.go

```diff
@@ -50,6 +50,19 @@ func Unmarshal(data []byte, dst encoding.Builder) error {
 	return copyGraph(dst, file.Graphs[0])
 }
 
+// UnmarshalMulti parses the Graphviz DOT-encoded data as a multigraph and
+// stores the result in dst.
```

Comment:
We should probably have a note that only the first graph of one can be stored (same for `Unmarshal`).

I'll file an issue.